frontend/src/modules/editor/components/FailedChecks.js

```diff
@@ -0,0 +1,65 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from 'fluent-react';
+
+import './FailedChecks.css';
+
+
+type Props = {|
+    errors: Array<string>,
+    warnings: Array<string>,
+    resetFailedChecks: () => void,
+|};
+
+
+/*
+ * Renders the failed checks popup.
+ */
+export default class FailedChecks extends React.Component<Props> {
+    closeFailedChecks = () => {
+        this.props.resetFailedChecks();
+    }
+
+    render() {
+        const { errors, warnings } = this.props;
+
+        if (!errors.length && !warnings.length) {
+            return null;
+        }
+
+        return <div className="failed-checks">
+            <Localized
+                id="editor-FailedChecks--close"
+                attrs={{ ariaLabel: true }}
+            >
+                <button
+                    aria-label="Close failed checks popup"
+                    className="close"
+                    onClick={ this.closeFailedChecks }
+                >×</button>
+            </Localized>
```

Comment:
@stasm Is there a way to make only the `ariaLabel` localized, without the button text?

Reply:
Sadly, not right now. There's an open issue about it: https://github.com/projectfluent/fluent.js/issues/184

Reply:
Thanks, that's a good enough answer right now.